src/darwin/Framework/CHIP/MTRCommissionableBrowser.mm

```diff
@@ -77,7 +81,10 @@ CHIP_ERROR Start(id<MTRCommissionableBrowserDelegate> delegate, MTRDeviceControl
         ResetCounters();
 
 #if CONFIG_NETWORK_LAYER_BLE
+        mBleScannerDelegateOwner = owner; // retain the owner until OnBleScanStopped is called
```

Comment:
What happens is Start is called twice ?

Reply:
That's prevented by the `VerifyOrReturnError` statements above.